apps/nextjs/src/components/ContextProviders/AnalyticsProvider.tsx

```diff
@@ -185,8 +185,8 @@ export const AnalyticsProvider: React.FC<AnalyticsProviderProps> = ({
   initAvo(
     { env: getAvoEnv(), ...avoOptions },
     {},
-    getAvoBridge({ posthog: posthogOak }),
-    getAvoBridge({ posthog: posthogAiBeta }),
+    getAvoBridge({ posthog: posthogOak, hubspot }),
+    getAvoBridge({ posthog: posthogAiBeta, hubspot }),
```

Comment:
I think this will double report to hubspot?

Reply:
I think you need something closer to

```suggestion
    getAvoBridge({ posthog: posthogOak }),
    getAvoBridge({ posthog: posthogAiBeta }),
    getAvoBridge({ hubspot }),
```

Reply:
Good point, I had a go like that but Avo init is not expecting a bridge. We could change that in Avo but it felt right to keep the analytics matching exactly what goes into the aila hubspot

Reply:
Ok yeah, this looks fine now